cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUtils.java

```diff
@@ -934,4 +935,13 @@ public static List<CompletionType> cachedValues() {
       return Collections.unmodifiableList(CACHED_VALUES);
     }
   }
+
+  private static Integer getNewPort() {
+    try {
+      return new ServerSocket(0).getLocalPort();
```

Comment:
Do you need to close the socket?

Reply:
We only use this in tests, so the socket is closing when the test finishes.